task/dumper/src/main.rs

```diff
@@ -92,7 +92,12 @@ impl idl::InOrderDumperImpl for ServerImpl {
         let mut nwritten = 0;
         let mut reg = 0;
 
-        let r = humpty::dump::<DumperError, 512, { humpty::DUMPER_EXTERNAL }>(
+        //
```

Comment:
**praise:** Nice comment to explain the magic number <3 Would a link to the great graph in #1804 also be worth it?

I presume the extra empty comment lines before and after are intended as well?

Reply:
Folks can find that in the issue -- I really like to leave links in source code to an absolute minimum because they often become dead over time.